.buildkite/test-rayjob-sample-yamls.yml

```diff
@@ -60,7 +60,9 @@
 
     # Install python 3.10 and pip
     - apt-get update
-    - apt-get install -y python3.10 python3-pip
```

Comment:
`apt-get install -y python3-pip` will report ["error: externally-managed-environment"](https://stackoverflow.com/questions/75608323/how-do-i-solve-error-externally-managed-environment-everytime-i-use-pip3) after I upgrade the image to `golang:1.19`.